helm/cluster-cloud-director/templates/cilium-helmrelease.yaml

```
@@ -34,7 +34,7 @@ spec:
       mode: kubernetes
     k8sServiceHost: api.{{ include "resource.default.name" $ }}.{{ .Values.global.connectivity.baseDomain }}
     k8sServicePort: "6443"
-    kubeProxyReplacement: strict
+    kubeProxyReplacement: true
```

I think this is a string
```suggestion
    kubeProxyReplacement: 'true'
```

---

Its not in the schema but previously this needed to be a string because stuff like 'strict' was supported. The helm template calls `toString` so a plain boolean should also be cool.

---

oh yes, although tests are passing, I'll keep it consistent with the cluster chart